tests/without_layout/test_pages.py

```
@@ -0,0 +1,64 @@
+        # -- No root layout.
+        (
+            "without_layout/inner?no_root_layout=true",
+            "GET",
+            RenderedPage.no_root_layout,
+        ),
+        (
+            "without_layout/inner?no_root_layout=true",
+            "POST",
+            RenderedHandleSubmit.no_root_layout,
+        ),
+    ),
```

Ruff (0.14.1)

Wrong values type in `pytest.mark.parametrize` expected `list` of `tuple`

Use `list` of `tuple` for parameter values

(PT007)